src/Microsoft.DotNet.Darc/src/Darc/Options/VirtualMonoRepo/VmrCommandLineOptions.cs

```diff
@@ -3,25 +3,47 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
-using System.Collections.Generic;
+using System.IO;
 using CommandLine;
+using Microsoft.DotNet.Darc.Helpers;
 using Microsoft.DotNet.DarcLib;
+using Microsoft.DotNet.DarcLib.Helpers;
+using Microsoft.DotNet.DarcLib.VirtualMonoRepo;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Extensions.Logging;
 
 namespace Microsoft.DotNet.Darc.Options.VirtualMonoRepo;
 
 internal abstract class VmrCommandLineOptions : CommandLineOptions
 {
-    [Value(0, Required = true, HelpText = 
-        "Repository names in the form of NAME or NAME:REVISION where REVISION is a commit SHA or other git reference (branch, tag). " +
-        "Omitting REVISION will synchronize the repo to current HEAD. Pass 'all' to update all repositories.")]
-    public IEnumerable<string> Repositories { get; set; }
-
-    [Option("vmr", Required = false, HelpText = "Path to the VMR; defaults to git root above cwd.")]
+    [Option("vmr", Required = true, HelpText = "Path to the VMR; defaults to git root above cwd.")]
```

Comment:
`; defaults to git root above cwd` this sounds unclear for me. Could we rephrase this a little bit.

Reply:
Fixed